h2/src/main/org/h2/engine/ConnectionInfo.java

```diff
@@ -98,7 +98,8 @@ public ConnectionInfo(String u, Properties info) {
                 "IFEXISTS", "INIT", "FORBID_CREATION", "PASSWORD", "RECOVER", "RECOVER_TEST",
                 "USER", "AUTO_SERVER", "AUTO_SERVER_PORT", "NO_UPGRADE",
                 "AUTO_RECONNECT", "OPEN_NEW", "PAGE_SIZE", "PASSWORD_HASH", "JMX",
-                "SCOPE_GENERATED_KEYS", "AUTHREALM", "AUTHZPWD", "NETWORK_TIMEOUT"};
+                "SCOPE_GENERATED_KEYS", "AUTHREALM", "AUTHZPWD",
+                "NETWORK_TIMEOUT", "STATEMENT_TIMEOUT"};
```

Comment:
Why this setting is listed here? It can be included into JDBC connection URL?

Reply:
Sorry for my mistake. The change of ConnectionInfo.java is reverted.

And please let me try to implement `SetTypes.QUERY_TIMEOUT` for `STATEMENT_TIMEOUT
` ...